source/conf.py

```diff
@@ -39,8 +39,8 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
-master_doc = 'index'
```

Comment:
not sure if we can change this - Sphinx may expect this variable to be set (although I'm not sure)

https://www.sphinx-doc.org/en/master/usage/configuration.html